test/tests/rackhd20/test_rackhd20_api_workflows.py

```diff
@@ -69,5 +70,143 @@ def test_api_20_workflows_put_ipmi(self):
+                        "friendlyName": "bad REST call",
+                        "injectableName": "Task.Rest.GET",
+                        "implementsTask": "Task.Base.Rest",
+                        "options": {
+                            "url": "http://localhost:8080/api/never/happen",
+                            "method": "GET"
+                        },
+                        "properties": {}
+                    }
+                },
+                {
+                    "label": "REST-call-final-good",
+                    "taskDefinition": {
+                        "friendlyName": "final REST call",
+                        "injectableName": "Task.Rest.GET",
+                        "implementsTask": "Task.Base.Rest",
+                        "options": {
+                            "url": "http://localhost:8080/api/1.1/nodes",
+                            "method": "GET"
+                        },
+                        "properties": {}
+                    },
+                    "waitOn": {
+                        "anyOf": {
+                            "REST-call-good": "succeeded",
+                            "REST-call-bad": "succeeded"
+                        },
+                        "REST-call-2": "succeeded"
+                    }
+                },
+                {
+                    "label": "REST-call-final-pending",
+                    "taskDefinition": {
+                        "friendlyName": "final REST call",
+                        "injectableName": "Task.Rest.GET",
+                        "implementsTask": "Task.Base.Rest",
+                        "options": {
+                            "url": "http://localhost:8080/api/1.1/nodes",
+                            "method": "GET"
+                        },
+                        "properties": {}
+                    },
+                    "waitOn": {
+                        "anyOf": {
+                            "REST-call-good": "failed",
+                            "REST-call-bad": "failed"
+                        }
+                    }
+                },
+                {
+                    "label": "REST-call-final-succeeded-2",
+                    "taskDefinition": {
+                        "friendlyName": "final REST call",
+                        "injectableName": "Task.Rest.GET",
+                        "implementsTask": "Task.Base.Rest",
+                        "options": {
+                            "url": "http://localhost:8080/api/1.1/nodes",
+                            "method": "GET"
+                        },
+                        "properties": {}
+                    },
+                    "waitOn": {
+                        "anyOf": {
+                            "REST-call-good": "failed",
+                            "REST-call-bad": "succeeded"
+                        }
+                    }
+                }
+            ]
+        }
+
+        node_id = fit_common.node_select()
```

Comment:
NODECATALOG = fit_common.node_select() at the start of this script performs the same thing.